src/aws.py

```diff
@@ -33,27 +33,6 @@ def stack_list(region=None):
 #
 #
 
-@task
```

Comment:
you got rid of the `create_ami` task? granted we haven't used it in the new builder, but should still work ...